widget/ui/src/components/Button/Button.styles.tsx

```diff
@@ -549,10 +549,9 @@ const ripple = keyframes({
 
 export const RippleContainer = styled('div', {
   position: 'absolute',
-  top: '0',
-  right: '0',
-  bottom: '0',
-  left: '0',
+  inset: '0',
+  overflow: 'hidden',
+  borderRadius: '100%',
```

Comment:
The ripple effect isn't functioning correctly :
![image01](https://github.com/rango-exchange/rango-client/assets/120931880/f41286bd-2fc7-48c3-a7f5-92fe72635aba)